tests/test_lithium.py

```diff
@@ -128,6 +137,40 @@ def __exit__(self, exc_type, exc_value, tb):  # pylint: disable=missing-return-d
             return _AssertLogsContext(self, logger, level)
 
 
+class DisableWER(object):
+    """Disable Windows Error Reporting for the duration of the context manager.
+
+    ref: https://msdn.microsoft.com/en-us/library/bb513638.aspx
+    """
+    # pylint: disable=too-few-public-methods
+
+    def __init__(self):
+        self.wer_disabled = None
+        self.wer_dont_show_ui = None
+
+    def __enter__(self):
+        if winreg is not None:
+            wer = winreg.OpenKey(winreg.HKEY_CURRENT_USER,
+                                 r"Software\Microsoft\Windows\Windows Error Reporting", 0,
+                                 winreg.KEY_QUERY_VALUE | winreg.KEY_SET_VALUE)
+            self.wer_disabled = bool(winreg.QueryValueEx(wer, "Disabled")[0])
+            if not self.wer_disabled:
+                winreg.SetValueEx(wer, "Disabled", 0, winreg.REG_DWORD, 1)
```

Comment:
As discussed over IRC, let's add a comment that we are not sure if we are disabling reporting to Microsoft, or whether we are disabling crash report generation altogether.